main.go

```diff
@@ -301,7 +305,13 @@ aliases:
 
 func getTagsFor(url string) ([]string, error) {
 	//TODO make a custom client with a small timeout!
-	res, err := http.Get(url)
+	skipTLS := os.Getenv("ALLOW_INSECURE") == "true"
```

Comment:
Can you make this lowercase so it would be more permissive? Allow TRUE and true.
Thanks!

Reply:
changed @bgadrian